jina/clients/base/__init__.py

```diff
@@ -37,6 +37,11 @@ def __init__(
         else:
             self.args = parse_client(kwargs)
 
+        # remove unused prefetch argument from self.args since the argument is preserved for backward compatibility
+        # the acutual value will be provided by the responsible method argument
+        if hasattr(self.args, 'prefetch'):
```

Comment:
I do not think this should be removed. It should be used in case the `post` one is not provided

Reply:
I didn't think about setting or enabling a global `prefetch` on the client. Since the arg is already there, I will refactor to include but it will complicate the logic in places where `**(self.args)` is used when invoking other client methods.